docs/layers/tile-layer.md

```diff
@@ -126,39 +128,163 @@ Hide tiles when under-zoomed.
 - Default: 0
 
 
-##### `maxCacheSize` (Number|Null, optional)
+##### `maxCacheSize` (Number, optional)
+
+The maximum number of tiles that can be cached. The tile cache keeps loaded tiles in memory even if they are no longer visible. It reduces the need to re-download the same data over and over again when the user pan/zooms around the map, providing a smoother experience.
 
-The maximum cache size for a tile layer. If not defined, it is calculated using the number of tiles in the current viewport times multiplied by `5`.
+If not supplied, the `maxCacheSize` is calculated as `5` times the number of tiles in the current viewport.
 
 - Default: `null`
 
 
-##### `strategy` (Enum, optional)
+##### `maxCacheByteSize` (Number, optional)
+
+The maximum memory used for caching tiles. If this limit is supplied, `getTileData` must return an object that contains a `byteLength` field.
+
+- Default: `null`
+
 
-How the tile layer determines the visibility of tiles. One of the following:
+##### `refinementStrategy` (Enum, optional)
+
+How the tile layer refines the visibility of tiles. One of the following:
 
 * `'best-available'`: If a tile in the current viewport is waiting for its data to load, use cached content from the closest zoom level to fill the empty space. This approach minimizes the visual flashing due to missing content.
 * `'no-overlap'`: Avoid showing overlapping tiles when backfilling with cached content. This is usually favorable when tiles do not have opaque backgrounds.
+* `'never'`: Do not display any tile that is not selected.
 
 - Default: `'best-available'`
 
 
-##### `tileToBoundingBox` (Function, optional)
+##### `Tileset` (Function, optional)
 
-**Advanced** Converts from `x, y, z` tile indices to a bounding box in the global coordinates. The default implementation converts an OSM tile index to `{west: <longitude>, north: <latitude>, east: <longitude>, south: <latitude>}`.
+A custom implemetation of the `Tileset2D` interface. This can be used to override some of the default behaviors of the `TileLayer`. See the `Tileset2D` documentation below.
+
+
+### Render Options
+
+##### `renderSubLayers` (Function, optional))
+
+Renders one or an array of Layer instances with all the `TileLayer` props and the following props:
+
+* `id`: An unique id for this sublayer
+* `data`: Resolved from `getTileData`
```

Comment:
Nit: Not something that bothers me, just calling out (and I think you already know) that in 3d tiles, the naming is "reversed":

- `data` is called `tile` (aka `tile content`)
- `tile` is called `tile header` 

I personally think the 3d tiles naming ultimately is less confusing as various tile loaders return the actual "tile".

Reply:
We cannot make a breaking API change this in 8.x.